onnx/defs/function.cc

```diff
@@ -57,6 +57,16 @@ Status FunctionBuilderRegistry::GetFunctions(
     }
 
     CheckerContext ctx;
+    std::unordered_map<std::string, int> op_set;
+    auto version_range =
+        OpSchemaRegistry::DomainToVersionRange::Instance().Map().at(
+            func_builder.GetDomain());
+    op_set.insert({func_builder.GetDomain(),
+                   function_proto->since_version() < version_range.second
```

Comment:
if function's since version is greater than the version_range.second, it should be an error, I think.